Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -58,59 +58,6 @@ class ApolloCodegenTests: XCTestCase {
     ])
   }
   
-  func testCreatingOptionsWithAllParameters() throws {
```

Comment:
I don't think we should kill this test, just change it to use the typescript engine for now.

Reply:
Yes - my bad. The option was incrementally added to the test vs. created the test for it.